cluster/manager.go

```diff
@@ -348,6 +354,7 @@ func (c *MemberlistManager) SetState(state NodeState) {
 	node := c.members[c.nodeName]
 	node.State = state
 	node.Updated = time.Now()
+	node.StateChange = time.Now()
```

Comment:
can you add comments to the HTTPNode definition about what Updated and StateChange represent / how they are used? it's not clear how they differ

Reply:
really?   one is set when the httpNode is `updated` one is set when the httpNodes's state changes

Reply:
yes, really.
It's more about *why* things are implemented this way. i.e. the overarching design of the code.
there is a lot of information in your head that is not there for others reading your code.

for example:
* why is StateChange never read from in the code, is it only for inspection by admin?
* why does changing the State and StateChange fields not trigger a gossip broadcast whereas changing Update would? (error clause in `MemberlistManager.NotifyUpdate`)
* who is responsible for updating the fields: the local nodes emitting their state, or the peers receiving it? (seems to be both?)
* does "update" include an update to the state field? seems to be yes except in the above mentioned exception case?

documentation helps in answering these sort of questions.

BTW should `SingleNodeManager.SetState` not update StateChange ?